scripts/migrate_user_id_to_integer.py

```
@@ -0,0 +1,127 @@
+            
+            if existing_user_ids:
+                logger.info(f"✅ 기존 user_id 값들: {[row.user_id for row in existing_user_ids]}")
+                
+                # 모든 값이 정수로 변환 가능한지 확인
+                for row in existing_user_ids:
+                    try:
+                        int(row.user_id)
+                        logger.info(f"   ✅ '{row.user_id}' → {int(row.user_id)} 변환 가능")
+                    except ValueError:
+                        logger.error(f"   ❌ '{row.user_id}'는 정수로 변환할 수 없습니다!")
+                        raise ValueError(f"user_id '{row.user_id}'를 정수로 변환할 수 없습니다.")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)